infrastructure/main.tf

```diff
@@ -187,33 +172,3 @@ module "db-v15" {
   pgsql_storage_mb     = var.pgsql_storage_mb
   force_user_permissions_trigger = "1"
 }
-
-resource "azurerm_key_vault_secret" "POSTGRES-USER-V15" {
```

Comment:
Are you intentionally deleting the v15 db keyvault entries as part of the v11 deletion?

Reply:
Yes. As the values of V15 DB are copied in to the existing key-vault/secrets.